debian/postinst

```diff
@@ -112,6 +112,7 @@ sed -i '/\(^\|#\)\s*unqualified-search-registries\s*=\s*/c\unqualified-search-re
 sed -i 's/\(#\|\^\)\s*\([^=]\+\)=\(suspend\|hibernate\)\s*$/\2=ignore/g' /etc/systemd/logind.conf
 sed -i '/\(^\|#\)MulticastDNS=/c\MulticastDNS=no' /etc/systemd/resolved.conf
 sed -i 's/\[Service\]/[Service]\nEnvironment=SYSTEMD_LOG_LEVEL=debug/' /lib/systemd/system/systemd-timesyncd.service
+sed -i "s/\.debian\./\./g;s/#FallbackNTP=/FallbackNTP=/"  /etc/systemd/timesyncd.conf
```

Comment:
This removes .debian from the ntp hostnames, so we don't needlessly dox our distro to DNS and NTP servers